source/grammar/index.rst

```diff
@@ -80,53 +88,59 @@ repetition is denoted by curly brackets ``+`` (one or more) and ``*`` (zero or m
     incrementor: "++" | "--
     pragma: "#pragma {" ... "}"
     declaration_statement: (quantum_declaration | classical_declaration | constant_declaration) ";"
-    assignment_expression: expression assignment_operator expression
-    binary_operator: "+" | "-" | "*" | "/" 
+    assignment_expression: "=" expression
+    binary_operator: "+" | "-" | "*" | "/"
```

Comment:
What is this trying to change/why? We need to handle assignments under all `assignment_operator`s below.